rust/kernel/print.rs

```diff
@@ -439,3 +439,31 @@ macro_rules! pr_cont (
         $crate::print_macro!($crate::print::format_strings::CONT, true, $($arg)*)
     )
 );
+
+/// Equivalent to [`std::dbg`](https://doc.rust-lang.org/std/macro.dbg.html), but uses `pr_debug`.
+#[macro_export]
+macro_rules! dbg {
+    // Comment from original / std implementation:
+
+    // NOTE: We cannot use `concat!` to make a static string as a format argument
+    // of `eprintln!` because `file!` could contain a `{` or
+    // `$val` expression could be a block (`{ .. }`), in which case the `eprintln!`
+    // will be malformed.
+    () => {
+        $crate::pr_debug!("[{}:{}]", core::file!(), core::line!());
```

Comment:
Note that `pr_*` does not add a newline -- did you try it at runtime?

Reply:
It seems to not matter, the prints show up in their own line. Maybe its just dmesg being nice though, I can definetely add the newline for good practice.

I messed up a bit: While now testing this again (we used to have it with `pr_info` back at https://github.com/niklasmohrin/linux/blob/2d803fceab31195ff6fa8ad209b2fe0438f130cf/rust/kernel/print.rs), I saw that printing on the debug level is actually different from the other levels (I found https://stackoverflow.com/questions/28936199/why-is-pr-debug-of-the-linux-kernel-not-giving-any-output). When I copied it over from our branch and saw that there now is a `pr_debug`, I just replaced it with that thinking it would be more fitting. Now I find `pr_debug` to actually be rather unconvenient though and would prefer something like `pr_info` (should have checked again before opening, sorry for that).

If I (as a dev just wanting to print debug info) would have to set up the kernel with some debug flags or settings, I would be more inclined to just use `pr_info` with mediocre formatting instead of `dbg`. Maybe I am missing something though ... :)

I guess there could be a CI job that ensures that there are no `dbg!` calls in production code so that the `info` level log is not getting "spammed" if that would be a problem. What do you or others think?

Reply:
> I guess there could be a CI job that ensures that there are no `dbg!` calls in production code so that the `info` level log is not getting "spammed" if that would be a problem. What do you or others think?

Sounds like a good idea -- could you please create an issue to about forgetting about it?